.github/workflows/release-docs.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
       - name: Move old latest release documentation folder
         if: ${{ steps.folder.outputs.migrate }}
-        run: mv -fv "${{ inputs.page_dir }}${{ inputs.docs_dir }}/content/en/docs/." "${{ inputs.page_dir }}${{ inputs.docs_dir }}/content/en/${{ steps.folder.outputs.migrate }}"
+        run: mv -fv "${{ inputs.page_dir }}${{ inputs.docs_dir }}/content/en/docs" "${{ inputs.page_dir }}${{ inputs.docs_dir }}/content/en/${{ steps.folder.outputs.migrate }}"
```

Comment:
I would go with cp -R here, as there is a potential risk that the device can be busy when moving. Afterwards we can use `rm -rf` for the cleanup as this removes the folder also when it's opened (device is busy)